charts/pulsar/values.yaml

```diff
@@ -410,6 +410,69 @@ zookeeper:
     usePolicy: true
     maxUnavailable: 1
 
+## Tiered Storage
+##
+storageOffload: {}
+  ## General
+  ## =======
+  # bucket: <bucket>
```

Comment:
I don't think `bucket` or `region` are general settings. They don't apply to filesystem offloader.